interface-definitions/load-balancing-inbound.xml.in

```diff
@@ -0,0 +1,233 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="load-balancing">
+    <properties>
+      <help>Configure load-balancing</help>
+      <priority>900</priority>
+    </properties>
+    <children>
+      <node name="inbound" owner="${vyos_conf_scripts_dir}/load-balancing-inbound.py">
```

Comment:
Maybe it will be better to rename "inbound" with the original "ipvsadm"?
If we decide to implement another pkg for balancing with inbound direction.

Reply:
Typically, LVS is a load-balancing component that fits into a router. It runs at L4 and is not a reverse proxy, but a true four-tier load balancer that uses some kind of routing and forwarding technique to load balance the inbound traffic for a particular VIP to the back end. No alternative to LVS has yet been found. Also, I don't think VYOS needs to use two different components at the same time to accomplish inbound load balancing

> Maybe it will be better to rename "inbound" with the original "ipvsadm"

Now consider that using component names instead of existing syntax may confuse users about what they are doing (users should not know exactly which component is doing, they just need to know what they are doing now, and can achieve a certain effect, without Need to know what is the component that implements the rule)

Reply:
For example one of them.
```
table ip lb {chain prerouting {type nat hook prerouting priority 0; policy accept;ip daddr 192.168.0.40 tcp dport http dnat nth 2 map {0: 192.168.100.10,1: 192.168.100.11}}}

nft add rule ip raw prerouting ip protocol tcp ip daddr set 192.168.1.100 tcp dport set 10 notrack
```
In any case, it's only my opinion. "Loadbalancing inbound" will give you no choice to another option and can confuse the user.

Reply:
@dmbaturin @c-po What are your thoughts?

Reply:
@sever-sever I've considered migrating from the "inbound" node to the "inbound ipvsadm" node

Please look at the submission, which is now complete